root/components/forms.tt

```diff
@@ -187,18 +187,48 @@
     [% END %]
 [%- END -%]
 
+[%- MACRO too_short_year_error(id) BLOCK -%]
+  <div class="error field-error" id="[% id %]" style="display: none">
+    [% WRAPPER form_row %]
```

Comment:
The row div here needs the `no-label` class added so that it's aligned properly, but you'll have to add the `<div class="row no-label">` here directly to do that, since the `form_row` macro doesn't allow it.